### PR TITLE
Instructions to install it from PyPI.

### PR DESCRIPTION
**Pull request by facundobatista**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,19 @@ There are multiple channels other than `stable`. See the full list with
 for everyday charming. With the snap you will always be up to date as
 Charmhub services and APIs evolve.
 
+You also can install it from PyPI but some system packages and a specific
```
Comment (jugmac00) [inline README.md:41]:
```suggestion
You also can install `charmcraft` from PyPI, but some system packages and a specific 
```

File: README.md [hunk 2]
```diff
@@ -38,6 +38,19 @@ There are multiple channels other than `stable`. See the full list with
 for everyday charming. With the snap you will always be up to date as
 Charmhub services and APIs evolve.
 
+You also can install it from PyPI but some system packages and a specific 
+Python module needs to be installed first: you need to check in the
```
Comment (jugmac00) [inline README.md:42]:
```suggestion
Python module need to be installed first: you need to check the 
```

File: README.md [hunk 3]
```diff
@@ -38,6 +38,19 @@ There are multiple channels other than `stable`. See the full list with
 for everyday charming. With the snap you will always be up to date as
 Charmhub services and APIs evolve.
 
+You also can install it from PyPI but some system packages and a specific 
+Python module needs to be installed first: you need to check in the 
+[Python APT library page](https://launchpad.net/ubuntu/+source/python-apt) 
+and find which source file you have to use according to your system (e.g. 
+for Impish it's `python-apt_2.2.1.tar.xz`). So the instructions (including 
+virtualenv creation and activation) would be:
```
Comment (jugmac00) [inline README.md:46]: I would leave out `(including virtualenv creation and activation)` as this is obvious when you look at the instructions, and it just makes the text harder to read.

File: README.md (hunk 1 above)
Comment (jugmac00) [reply README.md:41]: I would also write... "You can also install..." ... but I am not a native English speaker.

File: README.md (hunk 2 above)
Comment (jugmac00) [inline README.md:42]:
```suggestion
Python package needs to be installed first: you need to check in the 
```

File: README.md (hunk 1 above)
Comment (jugmac00) [reply README.md:41]: I would also remove `a specific` - I think it is clear, that you cannot install a random package :-)

File: README.md [hunk 4]
```diff
@@ -38,6 +38,19 @@ There are multiple channels other than `stable`. See the full list with
 for everyday charming. With the snap you will always be up to date as
 Charmhub services and APIs evolve.
 
+You also can install it from PyPI but some system packages and a specific 
+Python module needs to be installed first: you need to check in the 
+[Python APT library page](https://launchpad.net/ubuntu/+source/python-apt) 
+and find which source file you have to use according to your system (e.g. 
+for Impish it's `python-apt_2.2.1.tar.xz`). So the instructions (including
```
Comment (jugmac00) [inline README.md:45]:
```suggestion
and find which source file you have to use according to your system, e.g. 
for Impish it's `python-apt_2.2.1.tar.xz`. So the instructions (including 
```

**Review by jugmac00 — COMMENTED** [review]
Disclaimer: I am not a native Englisher speaker :-)

File: README.md [hunk 5]
```diff
@@ -38,6 +38,19 @@ There are multiple channels other than `stable`. See the full list with
 for everyday charming. With the snap you will always be up to date as
 Charmhub services and APIs evolve.
 
+You also can install it from PyPI but some system packages and a specific 
+Python module needs to be installed first: you need to check in the 
+[Python APT library page](https://launchpad.net/ubuntu/+source/python-apt) 
+and find which source file you have to use according to your system (e.g.
```
Comment (mr-cal) [inline README.md:44]: pedantic: 
~~find which source file you have to use according to your system~~ 
choose the source file that matches your system

**Review by mr-cal — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (facundobatista) [reply README.md:41]: Thanks, all three improvements are great.

File: README.md (hunk 2 above)
Comment (facundobatista) [reply README.md:42]: +1

File: README.md (hunk 3 above)
Comment (facundobatista) [reply README.md:46]: Simpler is better.

File: README.md (hunk 2 above)
Comment (facundobatista) [reply README.md:42]: +1

File: README.md (hunk 4 above)
Comment (facundobatista) [reply README.md:45]: I prefer with parenthesis.

File: README.md [hunk 6]
```diff
@@ -38,6 +38,19 @@ There are multiple channels other than `stable`. See the full list with
 for everyday charming. With the snap you will always be up to date as
 Charmhub services and APIs evolve.
 
+You also can install it from PyPI but some system packages and a specific 
+Python module needs to be installed first: you need to check in the 
+[Python APT library page](https://launchpad.net/ubuntu/+source/python-apt)
```
[the remark this answers is not in the document]
Comment (facundobatista) [reply README.md:43]: Improved. I mentioned the library generically, and then specified the instructions for Ubuntu.

**Pull request merged**